token/js/src/extensions/extensionType.ts

```diff
@@ -31,6 +33,12 @@ export enum ExtensionType {
     NonTransferableAccount,
     TransferHook,
     TransferHookAccount,
+    // ConfidentialTransferFee, // Not implemented yet
+    // ConfidentialTransferFeeAmount, // Not implemented yet
+    MetadataPointer = 18, // Remove number once above extensions implemented
+    // TokenMetadata, // Not implemented yet
+    // GroupPointer, // Not implemented yet
+    // TokenGroup, //Not implemented yet
```

Comment:
Yeah this works.

Tiny nit: There really isn't any reason to include any after `MetadataPointer` here. And on the next PR we just add `TokenMetadata`, no others.

Reply:
```suggestion
    // ConfidentialTransferFee, // Not implemented yet
    // ConfidentialTransferFeeAmount, // Not implemented yet
    MetadataPointer = 18, // Remove number once above extensions implemented
```

Reply:
Makes sense. Changed!